src/TimeTraveler.php

```diff
@@ -14,6 +14,9 @@ class TimeTraveler
      *
      * (new AbsoluteDate(2020-01-31))->modify('+1 month') = 2020-02-31 => 2020-03-02 so more than a month
      * addMonth(new AbsoluteDate(2020-01-31)) = 2020-02-29
+     *
+     * (new AbsoluteDate(2020-07-30))->modify('+1 month') = 2020-08-30 so less than a month
```

Comment:
Last day of July is July 31st and not 30th

Reply:
Good catch @hammat thanks!